apps/alpha-test-app/src/router/index.ts

```diff
@@ -0,0 +1,19 @@
+import type { RouteRecordRaw } from "vue-router";
+import HomeView from "../views/HomeView.vue";
+
+/** List of all routes the user can navigate to.
+ *  Please declare, whether authentication is enabled/disabled via the meta parameter.
+ */
+export const routes: RouteRecordRaw[] = [
+  {
+    path: "/",
+    name: "home",
+    component: HomeView,
+  },
+  {
+    path: "/layout-demo",
+    name: "layoutDemo",
+    component: () => import("../views/LayoutDemoView.vue"),
+    meta: { requiresAuth: false },
```

Comment:
```suggestion
    meta: { requiresAuth: false },
```

We don't have our Vue Blueprint here so this is not supported^^

Reply:
😅

Reply:
```suggestion
```